samples/copyclean.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#    This file is part of python-registry.
+#    It tries to make a clean copy of hive file
+#    clean meaning without unused (not understood) data
+#
```

Comment:
id have to see an example. there isnt any junk in a hive as determined by the header's hive length that i have seen. perhaps you are referring to data node records or slack in free cell data?